codec/src/main/java/io/netty/handler/codec/ConvertibleHeaders.java

```diff
@@ -90,13 +90,6 @@
     List<ConvertedType> getAllAndRemoveAndConvert(UnconvertedType name);
 
     /**
-     * Invokes {@link Headers#entries()} and lazily does a conversion on the results as they are accessed
```

Comment:
I think you are referring to `ConvertibleHeaders` overall?  Preferably it wouldn't be used/needed.  However some existing usages that assume `String` objects and use String type operations (regex, lastIndexOf, etc...).  We could push the translation on to the user, but the thought at the time was to preserve the interfaces's capabilities as much as possible...